components/screens/DailyRidesScreen/DailyRidesScreen.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Styles } from './DailyRidesStyleSheet';
+import { SafeAreaView, View, Text, FlatList, StatusBar, TouchableOpacity } from 'react-native';
```

Comment:
`View` and `StatusBar` are not being used and can be removed.

Reply:
Done